controllers/provisioners/ekscloudformation/ekscloudformation_test.go

```diff
@@ -1272,3 +1272,14 @@ func TestGetNodeAutoScalingGroupMetrics(t *testing.T) {
 	}
 }
 
+func TestParseCustomResourceYaml(t *testing.T) {
```

Comment:
Is the goal of this test to verify `ParseCustomResourceYaml` can handle an empty string?
Maybe we should rename the test to `TestParseCustomResourceYamlEmptyString`